Azure/azure-ad/_meta/fields.yml

```diff
@@ -13,7 +13,7 @@ action.target:
 azuread.Level:
   description: ''
   name: azuread.Level
-  type: long
+  type: keyword
```

Comment:
`azuread.Level` is not used in rules and alert-filter. I propose to remove it and use `log.level` instead. I was not able to find a mapping between int values and string representations.